### ISSUE TITLE
How to show date picker (calendar)?

### ISSUE THREAD

**Issue by Edilton** [globophobe/django-semantic-admin#20]
I can't put calendar as example in https://semantic-admin.com at field birthdate.
Is necessary an other action ?

Other question, exists suports other languages like Pt-br on calendar?

**Comment by globophobe** [comment]
Are you using the Django admin? It [should be automatic](https://github.com/globophobe/django-semantic-admin#usage) if you inherit from SemanticModelAdmin, or one of the Semantic Inline classes.

As for languages other than English, this is not currently but should be supported. I'll fix it on the weekend to [support setting the language](https://fomantic-ui.com/modules/calendar.html#language), using Django's js [translations for Pt-br](https://github.com/django/django/blob/628b6a686974698cbf820bad72f10dad133174ec/django/contrib/admin/locale/pt_BR/LC_MESSAGES/djangojs.po).

**Comment by Edilton** [comment]
Are you using the Django admin? It should be automatic if you inherit from SemanticModelAdmin, or one of the Semantic Inline classes.
A. Yes, I am using Django Admin. But I not inherit from SemanticModelAdmin, but from Django Default admin. That was the error. Thanks for help me.

As for languages other than English, this is not currently but should be supported. I'll fix it on the weekend to support setting the language, using Django's js translations for Pt-br.
A. That is a great news.

**Comment by globophobe** [comment]
Version 0.2.7 was released with support for translations in the calendar.

You can see it in the demo:

![multilingual-calendar](https://user-images.githubusercontent.com/891192/148680666-1b1b088f-341c-4034-9f4a-ea899b6a1813.png)

**Comment by Edilton** [comment]
That's great.

Is it possible edit time config too? I would like to set for dd/mm/YY.

**Comment by globophobe** [comment]
There is a [formatter](https://github.com/mdehoog/Semantic-UI-Calendar#settings) setting. I published a new release using:

```js
new Intl.DateTimeFormat({ dateStyle: 'short', timeStyle: 'short' }).format(date);
```

The result:

![intl-dateformat](https://user-images.githubusercontent.com/891192/149144031-842bb024-68c6-4baf-b3f1-6892c4a7921f.png)

**Comment by globophobe** [comment]
So, I realized that wasn't your request, and that I hadn't thought it through.

[Documentation for the calendar](https://globophobe.github.io/django-semantic-admin/calendar/ ) was added.

TLDR

1. Add the `javascript-catalog` to url_patterns
2. Add the `SEMANTIC_CALENDAR_OPTIONS`, to change the `intlDateTimeFormatOptions`

**Comment by globophobe** [comment]
Closing, as I believe the issue was resolved
